go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/google/uuid v1.3.0
 	github.com/google/wire v0.5.0
-	github.com/harness/lite-engine v0.5.103
+	github.com/harness/lite-engine v1.0.0-rc.137.0.20250407043243-5794fe017d1d
```

Comment:
Use the correct version of LE when that PR is merged.

Reply:
Updated